homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
```

Comment:
What do these entities do?
```py
    FroniusSensorEntityDescription(
        key="inverter_state",
        entity_category=EntityCategory.DIAGNOSTIC,
    ),
    FroniusSensorEntityDescription(
        key="error_code",
        entity_category=EntityCategory.DIAGNOSTIC,
    ),
    FroniusSensorEntityDescription(
        key="status_code",
        entity_category=EntityCategory.DIAGNOSTIC,
        entity_registry_enabled_default=False,
    ),
```

Reply:
- First is, according to API specs,
  > Status string reflecting the operational state of the inverter. Only provided for GEN24 since version 1.13.5

  I don't have such a device, so not sure what exactly this reports. Only see it set to "Running" in the fixtures I collected from users.

- Second shows a numerical error code - same as can be read on the small LCD display of the inverter.

- Third shows a numerical status code the inverter reports. There is a second entity translating that to a string status message.

Reply:
I would do the same approach here, try to do some data collection so we can end up with a better end result for the user

Reply:
error code translation addressed in #133394